File: IOT/anshulgurjar53_AnshulSinghGurjar_2024it1193_2/FunctionAndArrays/AnyBaseToDecimal.java
import java.util.*;
  
  public class Main{
  
  public static void main(String[] args) {
      Scanner scn = new Scanner(System.in);
      int n = scn.nextInt();
      int b = scn.nextInt();
      int d = getValueIndecimal(n, b);
      System.out.println(d);
   }
  
   public static int getValueIndecimal(int n, int b){
      // write your code here
      int c=0,s=0;
      while(n>0)
      {
          int a=n%10;
          s=(int)(s+a*Math.pow(b,c++));
          n/=10;
      }
      return s;
   }
  }

File: ML/niv168_VidhiSinha_2024cse1070_2/FunctionAndArrays/AnyBaseToDecimal.cpp
#include<iostream>
#include<math.h>
using namespace std;
int AnyToDec(int n, int b) {
//write your code here.
int d,c,m;
c=m=0;
while(n>0)
{
    d=n%10;
    c=c+(d*pow(b,m));
    n=n/10;
    m=m+1;
}
return c;
}
int main() {
  int n;
  int b;
  cin >> n;
  cin >> b;
  int res = AnyToDec(n, b);
  cout << res << endl;
}